inventory/os.cf

```diff
@@ -130,5 +130,14 @@ amzn_2::
      string => "Amazon 2",
      meta => { "inventory", "attribute_name=OS" };
 
+any::
+  "description"
+    string => "$(sys.os_release[PRETTY_NAME])",
+    if => and(
+      strcmp("$(sys.os_name_human)", "Unknown"),
+      isvariable("sys.os_release[PRETTY_NAME]")
+    ),
+    meta => { "inventory", "attribute_name=OS", "derived-from=sys.os_release" };
+
```

Comment:
yes, locally I have added

```
darwin::
  "product_name" string => execresult("/usr/bin/sw_vers --productName", useshell);
  "product_version" string => execresult("/usr/bin/sw_vers --productVersion", useshell);
  "description" string => "$(product_name) $(product_version)",
    meta => { "inventory", "attribute_name=OS", "derived-from=sw_vers command" };
```

But something that detects MacOS should likely go into the C code instead as we discussed.

Reply:
am testing this change locally. Thanks!